docs/markdown/Machine-files.md

```diff
@@ -84,21 +121,76 @@ command line will override any options in the native file. For example, passing
 
 In addition to special data that may be specified in cross files, this
 section may contain random key value pairs accessed using the
-`meson.get_external_property()`
+`meson.get_external_property`, or `meson.get_cross_property`.
 
-## Properties
+*Changed in 0.55.0* putting `<lang>_args` and `<lang>_link_args` in the
+properties section has been deprecated, and should be put in the built-in
+options section.
 
-*New for native files in 0.54.0*
+### Project specific options
 
-The properties section can contain any variable you like, and is accessed via
-`meson.get_external_property`, or `meson.get_cross_property`.
+*New in 0.55.0*
+
+Path options are not allowed, those must be set in the `[paths]` section.
+
+Being able to set project specific options in a cross or native file can be
+done using the `[project options]` section of the specific file (if doing a
+cross build the options from the native file will be ignored)
+
+An incomplete list of options is:
+- pkg_config_path
+- cmake_prefix_path
```

Comment:
Those are not project options, they are built-in, right?

Reply:
That looks like something went wrong in a rebase, it's in the wrong section and the paragraph before it about separate host vs build options are missing.